accord-core/src/main/java/accord/local/CommandStore.java

```diff
@@ -91,12 +90,17 @@ public int generation()
         return generation;
     }
 
-    public boolean hashIntersects(RoutingKey key)
+    public boolean hashIntersects(SortableKey key)
```

Comment:
Sortable is non-specific? This has to be either a Seekable or a Routable right? If it's the wrong kind then it won't intersect correctly?

Reply:
This intersection is consistent for both kinds of key. It's also hopefully not going to survive very long, as it's a source of inefficiency and complexity.

Reply:
If you intersect with a command store it's checking routable intersection not seekable intersection? I want to make sure I at least understand what is going on.